board/board.go

```diff
@@ -0,0 +1,102 @@
+package board
+
+import (
+	"fmt"
+)
+
+// Do not use interfaces everywhere
+type Board struct {
+	State [9]Cell
+}
+
+func (b *Board) MakeMove(i int, p rune) bool {
```

Comment:
Wouldn't it be better to rename `p` to `player` and make it clearer what that rune represents?

Reply:
Maybe. It's more of personal taste, than some global rules.
I usually use short names (especially if it used only in just 20-30 next lines).
It makes text (code) sparse and easier for me to capture some information out of it.